beacon_node/network/src/sync/block_lookups/mod.rs

```diff
@@ -345,72 +325,83 @@ impl<T: BeaconChainTypes> BlockLookups<T> {
             // we are already processing this block, ignore it.
             return;
         }
-
         let parent_lookup = ParentLookup::new(
             block_root,
             parent_root,
             peer_source,
             self.da_checker.clone(),
+            cx,
         );
-        self.request_parent_block_and_blobs(parent_lookup, cx);
+        self.request_parent(parent_lookup, cx);
     }
 
     /* Lookup responses */
 
-    pub fn single_block_lookup_response(
+    /// Get a single block lookup by its ID. This method additionally ensures the `req_counter`
+    /// matches the current `req_counter` for the lookup. This any stale responses from requests
```

Comment:
Missing a word here, I guess you meant 

```suggestion
    /// matches the current `req_counter` for the lookup. This ensures any stale responses from requests
```